.github/workflows/build_and_test.yaml

```diff
@@ -12,68 +12,54 @@ on:
       - "release-v*"
     paths-ignore:
       - "**/*.png"
-env:
-  GO_VERSION: 1.18.2
-  ENVOY_GATEWAY_DEV_IMAGE: envoyproxy/gateway-dev
-  ENVOY_GATEWAY_DEV_TAG: latest
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./tools/github-actions/setup-deps
+
+      # lint
+      - run: make lint-deps
```

Comment:
prefer `make lint` which handles install/deps for each linter under its target similar to what we have for kube tools

Reply:
The reason for adding a separate `make lint-deps` is so that when you view the logs for the `make lint` step, it shows just the actual linter output, so that you don't have to scroll through all of the output from installing the linters.